src/bundle/Resources/views/notifications/notifications_modal.html.twig

```diff
@@ -24,24 +24,16 @@
                 </button>
             </div>
             <div class="modal-body">
-                <table
-                    data-notifications="{{ path('ezplatform.notifications.render') }}"
-                    data-notifications-count="{{ path('ezplatform.notifications.count') }}"
-                    class="table n-table--notifications">
-                    <thead>
-                    <tr>
-                        <th>{{ 'notification.type'|trans|desc('Type') }}</th>
-                        <th>{{ 'notification.description'|trans|desc('Description') }}</th>
-                        <th>{{ 'notification.date'|trans|desc('Date') }}</th>
-                    </tr>
-                    </thead>
-                    <tbody class="n-table__body">
-                    {{ render(controller('EzPlatformAdminUiBundle:Notification:renderNotifications', {
-                        'limit': 5,
-                        'offset': 0,
+                <div class="n-notifications-modal__spinner n-notifications-modal__spinner--invisible">
```

Comment:
This is not `n-something` ... It should be `c-something` where `c` means component. Please, update all the HTML code for notifications.

Reply:
It was `n` as this was the `notifications-bundle`, now this should be `ez-notifications`.
Hard to tell if it's a BC break. I mean technically it is but I think we don't have a strong BC policy on templates.